client/extensions/woocommerce/state/ui/payments/methods/README.md

```diff
@@ -0,0 +1,58 @@
+Payment Methods
+================
+
+This module is used to manage payment method related UI.
+
+## Actions
+
+### `openPaymentMethodForEdit( siteId: number, id: string )`
```

Comment:
So, do we need to do both JSDoc and this format for everything? Seems like a waste to write 2x the documentation for the same code.